distribution/ddf-common/src/main/resources/etc/system.properties

```diff
@@ -284,7 +284,8 @@ karaf.secured.services=(&(osgi.command.scope=*)(osgi.command.function=*))
 #java.security.policy=${karaf.home}/etc/all.policy
 #org.osgi.framework.security=osgi
 #org.osgi.framework.trust.repositories=${karaf.home}/etc/trustStore.ks
-ws-security.subject.cert.constraints = .*
+#ws-security.subject.cert.constraints = pipe (|) separated list of regex patterns for DNs that are allowed to connect to the Security Token Service
```

Comment:
✏️  Nit: To be consistent with the other comments in the file, should we aviod putting the property name in the comment and put a space between the `#` and the description?